snmp/metadata.csv

```diff
@@ -211,6 +211,7 @@ snmp.sysGlobalTmmStatMemoryTotal,gauge,,byte,,[F5 BIG-IP] The total memory avail
 snmp.sysGlobalTmmStatMemoryUsed,gauge,,byte,,[F5 BIG-IP] The memory in use in bytes for TMM (Traffic Management Module).,0,snmp,
 snmp.sysMultiHostCpuIdle,gauge,,,,[F5 BIG-IP] The time spent by the specified processor doing nothing for the associated host.,0,snmp,
 snmp.sysMultiHostCpuIowait,gauge,,,,[F5 BIG-IP] The time spent by the specified processor waiting for external I/O to complete for the associated host.,0,snmp,
+snmp.sysMultiHostCpuUsageRatio,gauge,,fraction,,[F5 BIG-IP] This is usage ratio of CPU for the associated host.,0,snmp,
```

Comment:
```suggestion
snmp.sysMultiHostCpuUsageRatio,gauge,,,,[F5 BIG-IP] This is usage ratio of CPU for the associated host.,0,snmp,
```

Should I fix this too?

Other metrics which has the suffix `Ratio` doesn't have fraction as an `unit_name`.

https://github.com/kei6u/integrations-core/blob/29be2df08acbd72be6a3428d9f972d32adfaabdf/snmp/metadata.csv#L527-L529

Reply:
Yes, indeed, in this case, since the ratio is already between 0-100, using `percent` as unit is more appropriate.

(using `fraction` will *100 in UI, which is not necessary here)

After this change https://github.com/DataDog/integrations-core/pull/10924/commits/29be2df08acbd72be6a3428d9f972d32adfaabdf

1/ You will need to move `sysMultiHostCpuUsageRatio` to gauge section https://github.com/DataDog/integrations-core/pull/10924/files#diff-e4dc39d15fc0877866f30b10ae1169d5f29970f231d970c5c6c14dd571084f7eR262-R277

2/ There is probably a need to `513` to `515` since we are using `gauge` type now: https://github.com/DataDog/integrations-core/pull/10924/files#diff-0e66175c21a6949686b79d4f2a8acf1e473634da2638eb55a098f55445be66f3L50-R67
In the test, the check/integration is ran 2 times, and there are 2 cpus, so, initial 511 + 4 = 515.

Previously, using rate, the check/integration is also ran 2 times, but it will only produce 1 metric per cpu, 511 + 2 = 513. (that's because `rate` is a delta between 2 points)

Reply:
```suggestion
snmp.sysMultiHostCpuUsageRatio,gauge,,percent,,[F5 BIG-IP] This is usage ratio of CPU for the associated host.,0,snmp,
```

Reply:
For 1, moved it to gauge section. I thought the gauge section is [here](https://github.com/DataDog/integrations-core/blob/8a0ec3f150e78768ac7eb4a95a68491962774a1d/snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml#L56-L95) because there are `forced_type: gauge`. Does it make sense to you? abeb254

For 2, c4e1201
And fix `unit_name` to percent. c4e1201

Reply:
For 1, sorry if I wasn't clear, I was referring to moving `sysMultiHostCpuUsageRatio` to gauge section in test file here https://github.com/DataDog/integrations-core/blob/c4e12013d1b5fdc66267736f9468ea4f472d115e/snmp/tests/test_profiles.py#L262-L277 
Otherwise the tests might not pass.

Reply:
Thanks for clarifying it.
moved `sysMultiHostCpuUsageRatio` to sysMultiHostCpuUsageRatio. c493c52

Reply:
Unit tests are failing, looks like we need this instead:

```
    cpu_gauges = [
        'sysMultiHostCpuUsageRatio',
    ]

[...]

    for metric in cpu_gauges:
        aggregator.assert_metric('snmp.{}'.format(metric), metric_type=aggregator.GAUGE, tags=['cpu:0'] + tags, count=1)
        aggregator.assert_metric('snmp.{}'.format(metric), metric_type=aggregator.GAUGE, tags=['cpu:1'] + tags, count=1)
```

Reply:
I appreciate your reviews!
Added your suggestions to the test. 2a544d4

Reply:
👍 , you need to remove `sysMultiHostCpuUsageRatio` added previously to `gauges` list: https://github.com/DataDog/integrations-core/pull/10924/files#diff-e4dc39d15fc0877866f30b10ae1169d5f29970f231d970c5c6c14dd571084f7eR277

Reply:
ah, I should've removed it. Thanks! 3c67db7

Reply:
And I should've checked the ci workflows. I'm sorry for asking you to review it so many times.

Reply:
e2e failed again...
I am not sure how to fix this.
I'm sorry, but could you please help me?
https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=86170&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=57642e18-890a-5eef-b695-2b3c448c1261&l=345

Reply:
No worry, the failing test is unrelated to this PR. It's a flaky test that fail sometimes, we are trying to fix it (working on it).

I just retried the CI job. I think the PR is ready to be merged.